Security/Fundamentals/Security_Principles.mediawiki

```diff
@@ -169,9 +175,10 @@ Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/mas
 
 '''Do'''
 * Use credentials-based authentication and user session management where the session information is passed by the user (https://research.google.com/pubs/pub44860.html)
+* Strongly authenticate access to any resource that needs to be protected.
 
 '''Do not'''
 * Trust traffic from a certain network address.
 * Use IP ACLs to control access to systems.
-* Trust the office network for access to devices.
+* Trust the office network for access to management devices and administration interfaces.
```

Comment:
I think all of these "do not" are in the context of "do not use X as the sole authentication mechanism". I don't where best to add that context, though.